src/components/sync-sites-modal-selector.tsx

```diff
@@ -290,7 +320,8 @@ function SiteItem( {
 						variant="link"
 						onClick={ () => getIpcApi().openURL( `https://wordpress.com/plans/${ site.id }` ) }
 					>
-						{ __( 'Upgrade plan ↗' ) }
+						{ __( 'Upgrade plan' ) }
```

Comment:
Is this change intentional?

Reply:
Yes, I did this on purpose to use the component instead of the arrow included in the translate string. We are using both approaches in the codebase and I think the use of the component is cleaner.

Do you prefer having the arrow inside the string instead?

Reply:
Oh, I see the ArrowIcon now, thanks for the explanation.

Reply:
Anytime!